frontend/components/ClickableUrls/ClickableUrls.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import * as DOMPurify from "dompurify";
+import classnames from "classnames";
+
+interface IClickableUrls {
+  text: string;
+  className?: string;
+}
+
+const baseClass = "clickable-urls";
+
+const ClickableUrls = ({ text, className }: IClickableUrls): JSX.Element => {
+  const clickableUrlClasses = classnames(baseClass, className);
+
+  // Regex to find case insensitive URLs and replace with link
+  const replacedLinks = text.replaceAll(
```

Comment:
Shoot, I'm still seeing not all links being matched:
<img width="507" alt="Screenshot 2023-08-10 at 3 49 24 PM" src="https://github.com/fleetdm/fleet/assets/61553566/0c3046fd-abf0-46b6-ad27-a40547d6c06d">


I think this is what we want instead of lines 16-19:
`const replacedLinks = text.replaceAll(
    /(https?)?(:\/\/)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/g,
    '<a href="$&" target="_blank">$&</a> '
  );`

Which results in: 
<img width="542" alt="Screenshot 2023-08-10 at 3 45 58 PM" src="https://github.com/fleetdm/fleet/assets/61553566/12bda9de-e4e0-495f-b079-8c60124182ed">

Reply:
Ah, there's actually an issue with my suggestion as well – the last two links, without https nor http, are being appended to the current URL (e.g. `https://localhost:8080/hosts/96/example.co.`)